### PR TITLE
Revive old thrift and mark `in_flume` as obsoleted for Flume NG

### PR DESCRIPTION
**Pull request by cosmo0920**
Related to #10.

We should provide `in_flume` for Flume legacy users until 0.2.0 and display obsoleted message for Flume NG users.

### CODE REVIEW

File: lib/fluent/plugin/out_flume.rb [hunk 1]
```diff
@@ -32,8 +32,8 @@ class FlumeOutput < BufferedOutput
 
   def initialize
     require 'thrift'
+    require 'fluent/plugin/thrift/flume_types'
```
Comment (repeatedly) [inline lib/fluent/plugin/out_flume.rb]: Why is this require order changed?

Comment (cosmo0920) [inline lib/fluent/plugin/out_flume.rb]: Curiously, this change prevents the following error in test:

```log
E
=================================================================================================================================================================================================
Error: test_write(FlumeOutputTest): NameError: uninitialized constant Status
/Users/hhatake/Github/fluent-plugin-flume/lib/fluent/plugin/thrift/thrift_source_protocol.rb:88:in `<class:Append_result>'
/Users/hhatake/Github/fluent-plugin-flume/lib/fluent/plugin/thrift/thrift_source_protocol.rb:83:in `<module:ThriftSourceProtocol>'
/Users/hhatake/Github/fluent-plugin-flume/lib/fluent/plugin/thrift/thrift_source_protocol.rb:10:in `<top (required)>'
/Users/hhatake/Github/fluent-plugin-flume/lib/fluent/plugin/out_flume.rb:38:in `require'
/Users/hhatake/Github/fluent-plugin-flume/lib/fluent/plugin/out_flume.rb:38:in `initialize'
/Users/hhatake/Github/fluent-plugin-flume/vendor/bundle/ruby/2.3.0/gems/fluentd-0.12.20/lib/fluent/test/base.rb:44:in `new'
/Users/hhatake/Github/fluent-plugin-flume/vendor/bundle/ruby/2.3.0/gems/fluentd-0.12.20/lib/fluent/test/base.rb:44:in `initialize'
/Users/hhatake/Github/fluent-plugin-flume/vendor/bundle/ruby/2.3.0/gems/fluentd-0.12.20/lib/fluent/test/input_test.rb:28:in `initialize'
/Users/hhatake/Github/fluent-plugin-flume/vendor/bundle/ruby/2.3.0/gems/fluentd-0.12.20/lib/fluent/test/output_test.rb:51:in `initialize'
/Users/hhatake/Github/fluent-plugin-flume/test/plugin/test_out_flume.rb:16:in `new'
/Users/hhatake/Github/fluent-plugin-flume/test/plugin/test_out_flume.rb:16:in `create_driver'
/Users/hhatake/Github/fluent-plugin-flume/test/plugin/test_out_flume.rb:89:in `test_write'
     86:   def test_write
     87:     time = Time.parse("2011-12-21 13:14:15 UTC").to_i
     88: 
  => 89:     d = create_driver
     90:     d.emit({"k11"=>"v11", "k12"=>"v12"}, time)
     91:     d.emit({"k21"=>"v21", "k22"=>"v22"}, time)
     92:     d.run
=================================================================================================================================================================================================
```

Comment (repeatedly) [inline lib/fluent/plugin/out_flume.rb]: Hmm... okay.

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**